docs/installation/compose.md

```diff
@@ -0,0 +1,221 @@
+# Installation with docker compose
+
+We provide a sample configuration for running Docs using Docker Compose. Please note that this configuration is experimental, and the official way to deploy Docs in production is to use [k8s](../installation/k8s.md)
+
+## Requirements
+
+- A modern version of Docker and its Compose plugin.
+- A domain name and DNS configured to your server.
+- An Identity Provider that supports OpenID Connect protocol - we provide [an example to deploy Keycloak](../examples/compose/keycloak/README.md).
+- An Object Storage that implements S3 API - we provide [an example to deploy Minio](../examples/compose/minio/README.md).
+- A Postgresql database
+- A Redis database
+
+## Software Requirements
+
+Ensure you have Docker Compose(v2) installed on your host server. Follow the official guidelines for a reliable setup:
+
+Docker Compose is included with Docker Engine:
+
+- **Docker Engine:** We suggest adhering to the instructions provided by Docker
+  for [installing Docker Engine](https://docs.docker.com/engine/install/).
+
+For older versions of Docker Engine that do not include Docker Compose:
+
+- **Docker Compose:** Install it as per the [official documentation](https://docs.docker.com/compose/install/).
+
+> [!NOTE]
+> `docker-compose` may not be supported. You are advised to use `docker compose` instead.
+
+## Step 1: Prepare your working environment:
+
+```bash
+mkdir -p docs/env.d
+curl -o compose.yaml https://raw.githubusercontent.com/suitenumerique/docs/refs/heads/main/docs/examples/compose.yaml
```

Comment:
you put the compose.yaml in a different directory than docs/examples/compose.yaml